### PR TITLE
Solves issue #24

### PR DESCRIPTION
**Pull request by pipacsba**

### CODE REVIEW

File: repository.json
Comment (jeremywillans) [inline repository.json:1]: why are you updating the maintainer on this file?

**Comment by jeremywillans** [comment]
Ive added arch aarch64 to the instances directly.

File: repository.json
Comment (pipacsba) [reply repository.json:1]: I am sorry, this was my first pull request attempt, I only wanted to create pull requests for the config.yaml files.

**Comment by pipacsba** [comment]
thank you for the quick fix. I am checking it now.

**Pull request closed without merging**